x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/logic/get_rules_to_update.ts

```diff
@@ -65,6 +64,8 @@ export const mergeExceptionLists = (
       return latestPrebuiltRule;
     }
   } else {
+    // Carry over the previous version's exception list
+    latestPrebuiltRule.exceptions_list = installedRule?.params.exceptionsList;
```

Comment:
Should this logic be incorporated inside the `logic/detection_rules_client/methods/upgrade_prebuilt_rule.ts` method?

Reply:
Moved it over